neo/sys/posix/posix_main.cpp

```diff
@@ -688,11 +688,7 @@ Posix_Cwd
 const char* Posix_Cwd()
 {
 	static char cwd[MAX_OSPATH];
```

Comment:
oh one thing I just noticed: remove the `#define MAX_OSPATH` in this file, there's a `static const int MAX_OSPATH = 256;` in `neo/framework/FileSystem.h`, which should be set to a sane value for non-windows systems (see discussion above)